core/tests/performance_tests/splash_test.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2016 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for the splash page"""
+
+import unittest
+
+# pylint: disable=import-error, wrong-import-position
+from core.tests.performance_tests import base
+
+
+class SplashPagePerformanceTest(base.TestBase):
+    """Tests for the splash page."""
+
+    SPLASH_URL = 'http://localhost:8181/splash'
+
+    def setUp(self):
+        super(SplashPagePerformanceTest, self).setUp()
+
+    def test_splash_page_has_less_than_10_megabytes_sent_to_the_client(self):
+        """"Is total size below 10,000,000 in bytes."""
```

Comment:
This docstring probably isn't needed since the method name aptly explains what is being tested. Ditto for ``test_splash_page_loads_under_10_seconds``, ``SplashPagePerformanceForCachedStateTest.test_splash_page_has_less_than_1_megabytes_sent_to_the_client``, and ``SplashPagePerformanceForCachedStateTest.test_splash_page_loads_under_3_seconds``